src/to-fresh.js

```diff
@@ -71,6 +71,7 @@ Convert JRS resume sections to FRESH.
             start: edu.startDate,
             end: edu.endDate,
             grade: edu.gpa,
+            title: edu.title,
```

Comment:
This `title` field doesn't appear to exist in the [jsonresume schema](https://github.com/jsonresume/resume-schema/blob/v1.0.0/schema.json#L185).

Reply:
Thanks.  I decided to ignore the spec.  There are too many little pieces missing.

I'm thinking of writing my own minimal conversion tool that will use handlebars and accept whatever one puts in it.  The JSONRESUME standard appears like a good starting point for this tool.

BTW, there's another spec that sounds more promising to me because it's maintained by the European Union: http://interop.europass.cedefop.europa.eu/data-model/
It looks a bit complicated but I hope it covers everything that I need.

Reply:
That does look promising but complicated. Could be interesting to explore it more in the future.